source/operations/external-iam/configure-ad-ldap-external-identity-management.rst

```diff
@@ -150,99 +195,60 @@ An AD/LDAP user with no assigned policy *and* with membership in groups with no
+	    export MINIO_IDENTITY_LDAP_COMMENT="LDAP test server"
 
-         MinIO supports specifying the AD/LDAP provider
-         settings using :mc-conf:`configuration settings <identity_ldap>`. The 
-         :mc:`minio server` process applies the specified settings on its next
-         startup. For distributed deployments, the :mc:`mc admin config`
-         command applies the configuration to all nodes in the deployment.
+         For complete documentation on these variables, see :ref:`minio-server-envvar-external-identity-management-ad-ldap`
 
-         The following example code sets *all* configuration settings related to
-         configuring an AD/LDAP provider for external
-         identity management. The minimum *required* setting are:
-         
-         - :mc-conf:`identity_ldap server_addr <identity_ldap.server_addr>`
+      .. tab-item:: MinIO Console
 
-         - :mc-conf:`identity_ldap lookup_bind_dn <identity_ldap.lookup_bind_dn>`
+         MinIO supports specifying the AD/LDAP provider settings using the :ref:`MinIO Console <minio-console>`.
 
-         - :mc-conf:`identity_ldap lookup_bind_password <identity_ldap.lookup_bind_password>`
-         
-         - :mc-conf:`identity_ldap user_dn_search_base_dn <identity_ldap.user_dn_search_base_dn>`
-         
-         - :mc-conf:`identity_ldap user_dn_search_filter <identity_ldap.user_dn_search_filter>`
-
-         .. code-block:: shell
-            :class: copyable
-
-            mc admin config set ALIAS/ identity_ldap \
-               server_addr="ldaps.example.net:636" \
-               lookup_bind_dn="CN=xxxxx,OU=xxxxx,OU=xxxxx,DC=example,DC=net" \
-               lookup_bind_password="xxxxxxxx" \
-               user_dn_search_base_dn="DC=example,DC=net" \
-               user_dn_search_filter="(&(objectCategory=user)(sAMAccountName=%s))" \
-               group_search_filter= "(&(objectClass=group)(member=%d))" \
-               group_search_base_dn="ou=MinIO Users,dc=example,dc=net" 
-
-         For more complete documentation on these settings, see
-         :mc-conf:`identity_ldap`.
+	 .. include:: /includes/common-minio-external-auth.rst
+            :start-after: start-minio-ad-ldap-console-enable
+            :end-before: end-minio-ad-ldap-console-enable
 
    2) Restart the MinIO Deployment
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   You must restart the MinIO deployment to apply the configuration changes. 
+   You must restart the MinIO deployment to apply the configuration changes.
    Use the :mc-cmd:`mc admin service restart` command to restart the deployment.
 
    .. code-block:: shell
       :class: copyable
 
       mc admin service restart ALIAS
 
-   Replace ``ALIAS`` with the :ref:`alias <alias>` of the deployment to 
-   restart.
+   Replace ``ALIAS`` with the :ref:`alias <alias>` of the deployment to restart.
+
+   If you configured AD/LDAP from the MinIO Console, no additional action is required.
```

Comment:
Maybe move this up to just after the first line in case someone reads and executes in strict ordering here